src/lib/get-token.js

```diff
@@ -18,6 +19,43 @@ async function getToken(store, client, auth, state) {
     scope: getScopeForResponse(client, auth.scope)
   };
 
+  const scopes = auth.scope.split(' ');
+
+  if (scopes.includes('openid')) {
+    const provider = process.env.PROVIDER_URL || 'http://localhost:3020/';
+    const secret = process.env.SECRET || 'uber5';
+    const now = new Date();
+    const { identifiers } = await store.getUserById(auth.userId);
+    // eslint-disable-next-line one-var
+    let email, phone;
+    identifiers.forEach(identifer => {
+      if (identifer.startsWith('email')) {
+        const splitted = identifer.split(':');
+        [, email] = splitted;
+      }
+      if (identifer.startsWith('phone')) {
+        const splitted = identifer.split(':');
+        [, phone] = splitted;
+      }
```

Comment:
This is totally not DRY... you can fix this together with making the identifier keys dynamic, see comment above.